packages/nextjs/components/HooksExample.tsx

```diff
@@ -122,6 +153,45 @@ const HooksExample: React.FC = () => {
             )}
           </div>
         </div>
+        {/* ETH Balance */}
+        <div className="rounded-[5px] bg-base-100 border border-gradient p-4 relative shadow">
+          <div className="trapeze"></div>
+          <h2 className="text-xl font-semibold mb-4">
+            Contract ETH Balance (useScaffoldEthBalance)
+          </h2>
+          {isEthBalanceLoading ? (
+            <p>Loading contract...</p>
+          ) : contract ? (
+            <p>{ethBalance} ETH</p>
+          ) : (
+            <p>No contract loaded</p>
+          )}
```

Comment:
Would prefer to not have nested ternary ops like this since the code becomes confusing. You can use the suggestion below or use if statements.

```suggestion
          { isEthBalanceLoading &&  <Skeleton />}
          {!isEthBalanceLoading && contract ? (
            <p>{ethBalance} ETH</p>
          ) : (
            <p>No contract loaded</p>
          )}
```

Reply:
Hi @bitfalt,

Yes, the Skeleton component replaces the loading state and does so in a more stylish way than just displaying a simple text message like "Loading...". I think you can use it by importing `Skeleton` from `@radix-ui/themes`:

```javascript
import { Skeleton } from "@radix-ui/themes";
```

```javascript
{/* ETH Balance */}
  {isEthBalanceLoading && <Skeleton />}  
  {(!isEthBalanceLoading && contract) ? (  
    <p>{ethBalance} ETH</p>  
  ) : (  
    <p>No contract loaded</p>  
  )}  
</div>

{/* STRK Balance */}
  {isStrkBalanceLoading && <Skeleton />}
  {(!isStrkBalanceLoading && contract) ? (  
    <p>{strkBalance} STRK</p>  
  ) : (  
    <p>No contract loaded</p>  
  )}  
</div>
```

Reply:
okay, perfect. Thank you so much for explaining I'll make the changes :D